variantEffectPredictor.wdl

```diff
@@ -176,7 +176,7 @@ task chromosomeArray {
   }
 
   command <<<
-    zcat ~{vcfFile} | grep -v ^# | cut -f 1 | sort -V | uniq
+    zcat ~{vcfFile} | grep -v ^# | cut -f 1 | uniq
```

Comment:
I think sort -V -k1,1 should also work. 
I am approving the PR assuming that all the VCFs will be sorted by chromsome